python/tk_multi_workfiles/entity_models/deferred_model.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, entity_type, filters, hierarchy, fields, deferred_query, *arg
         self._deferred_query = deferred_query
         self._task_step_icons = {}
         self._deferred_models = {}
+        self._pending_delayed_data_refreshed = False
```

Comment:
i suggest adding a comment to briefly explain what state we are tracking. (It's not clear to be what *pending delayed data* is.)